tests/integration/ethernet_mtu_test.py

```diff
@@ -41,15 +41,15 @@ def eth1(eth1_up):
 
 @pytest.fixture
 def eth1_with_ipv6(eth1_up):
-    ifstate = eth1_up[schema.Interface.KEY][0]
-    ifstate[schema.Interface.IPV6][schema.InterfaceIPv6.ENABLED] = True
+    ifstate = eth1_up[Interface.KEY][0]
+    ifstate[Interface.IPV6][InterfaceIPv6.ENABLED] = True
     libnmstate.apply(eth1_up)
 
 
 def test_increase_iface_mtu():
     desired_state = statelib.show_only(('eth1',))
-    eth1_desired_state = desired_state[INTERFACES][0]
-    eth1_desired_state['mtu'] = 1900
+    eth1_desired_state = desired_state[Interface.KEY][0]
+    eth1_desired_state[Interface.MTU] = 1900
```

Comment:
Please mention in the commit message that you also used `Interface.MTU`.

Reply:
Done!